src/dbus_api/pool.rs

```diff
@@ -6,14 +6,15 @@ use std::{collections::HashMap, path::Path, vec::Vec};
 
 use dbus::{
     self,
-    arg::{Array, IterAppend},
+    arg::{Array, IterAppend, RefArg, Variant},
     tree::{
         Access, EmitsChangedSignal, Factory, MTFn, MethodErr, MethodInfo, MethodResult, PropInfo,
+        Tree,
     },
     Message,
 };
-
```

Comment:
This looks like a stray line add, I hope its not rustfmt again.

Reply:
This may be due to a nightly rustfmt, potentially the same as the `.clone()` change I made.

Reply:
Should I change this back?

Reply:
Yes, please do.

Reply:
Oh! I just looked at this more closely. The grouping of imports was inconsistent here so I was grouping all of the crate imports as one unit. I'll change it back if you don't like that.